src/Microsoft.Health.Fhir.Core/Features/Operations/ConvertData/BaseTokenVerifier.cs

```diff
@@ -0,0 +1,15 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+namespace Microsoft.Health.Fhir.Core.Features.Operations.ConvertData
+{
+    public class BaseTokenVerifier : IContainerRegistryTokenVerifier
```

Comment:
We should add some implementation in the OSS to check the identity for accessing the container registry.